<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorLine</class>
 <widget class="QWidget" name="ColorLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QRadioButton" name="radioButton">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
